### DISCUSSION TITLE
How to get Acceptance Criteria of WI

### DISCUSSION THREAD

**Discussion by JuanSGA24** [rfennell/AzurePipelines#1114] [Q&A] [answered]
Hello!

We are using your extension to create in automatic way the release notes. We want to show each US related to the PR and inside of each US we want to show the **Acceptance Criteria field**. We research about how to do it and we do not found information about this field.

```
# User Stories
{{#forEach this.workItems}}
## {{this.id}}**  {{lookup this.fields 'System.Title'}}
- **Description** {{{lookup this.fields 'System.Description'}}}
- **Acceptance Criteria** {{{lookup this.fields '(Acceptance Criteria field)'}}}
{{/forEach}}
```
This is the code that we have, and we do not know how to fill the **Acceptance Criteria field**.

The Acceptance Criteria field is in our Work Item as image below: 
https://gyazo.com/c8eddf260c867717ac94c743af3f3958

![image](https://gyazo.com/c8eddf260c867717ac94c743af3f3958)

**Comment by rfennell** [comment, marked as answer]
You need to enter the name not the description of the field.

You can find this 

- By dumping the JSON payload, as used by the template processor, for the task to a file and having a look for it in the fields section. There are parameters for the task to dump the JSON with the release notes file as a JSON file (my preferred option)  or to the console log (not usually a good idea as the payload can be large and so hard to read)
- By checking your process template, find the field and look for the name (see below). The field will be in the array as `Custom.BugGrading` (for my example)
![image](https://user-images.githubusercontent.com/324718/138856858-df4ebb8c-3dd3-4512-ae91-e22fd9ecbf65.png)

**Reply by rfennell** [reply]
Also see #970

**Reply by JuanSGA24** [reply]
Thanks for your quick reply! Yeah I printed the JSON object and I found that for the Acceptance Criteria the field was "Microsoft.VSTS.Common.AcceptanceCriteria".

Thank you so much for your help! 😄
